src/Fixer/Alias/NoAliasFunctionsFixer.php

```diff
@@ -245,13 +351,36 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
      */
     protected function createConfigurationDefinition()
     {
-        $sets = ['@internal', '@IMAP', '@mbreg', '@all', '@time', '@exif'];
+        $sets = [
+            '@all' => 'all listed sets',
+            '@internal' => 'native functions',
+            '@IMAP' => 'IMAP functions',
+            '@mbreg' => 'from `ext-mbstring`',
+            '@time' => 'time functions',
+            '@exif' => 'EXIF functions',
+            '@snmp' => 'SNMP functions',
+            '@ldap' => 'LDAP functions',
+            '@mysqli' => 'mysqli functions',
+            '@pg' => 'pg functions',
+            '@oci' => 'oci functions',
+            '@odbc' => 'odbc functions',
+            '@openssl' => 'openssl functions',
+            '@sodium' => 'libsodium functions',
+            '@ftp' => 'FTP functions',
+            '@posix' => 'POSIX functions',
+            '@pcntl' => 'PCNTL functions',
+        ];
+
+        $list = '';
+        foreach ($sets as $set => $description) {
+            $list .= "* `{$set}` ({$description})\n";
+        }
 
         return new FixerConfigurationResolver([
-            (new FixerOptionBuilder('sets', 'List of sets to fix. Defined sets are `@internal` (native functions), `@IMAP` (IMAP functions), `@mbreg` (from `ext-mbstring`) `@all` (all listed sets).'))
+            (new FixerOptionBuilder('sets', "List of sets to fix. Defined sets are:\n\n{$list}"))
                 ->setAllowedTypes(['array'])
-                ->setAllowedValues([new AllowedValueSubset($sets)])
-                ->setDefault(['@internal', '@IMAP'])
+                ->setAllowedValues([new AllowedValueSubset(array_keys($sets))])
+                ->setDefault(['@internal', '@IMAP', '@pg'])
```

Comment:
please don't change the default

Reply:
I'm not quite sure about this set: the only alias function contained in the `@pg` is the extremely old pg_exec alias, deprecated way back in PHP 4.
I'm not sure if it's worth removing from the defaults.